mods/voidmulti.sc2mod/base.sc2data/gamedata/actordata.xml

```diff
@@ -840,6 +858,9 @@
         <On Terms="Upgrade.AnabolicSynthesis.Remove; !ValidateUnit IsUltraliskBurrowed" Target="UltraliskAnabolicSynthesisUpgrade" Send="Destroy"/>
         <On Terms="AbilMorph.BurrowUltraliskUp.Finish; ValidatePlayer HasAnabolicSynthesisUpgrade" Send="Create UltraliskAnabolicSynthesisUpgrade"/>
         <On Terms="AbilMorph.BurrowUltraliskDown.Start" Target="UltraliskAnabolicSynthesisUpgrade" Send="Destroy"/>
+        <DeathActorModelLow value="UnitDeathModel"/>
+        <BarOffset value="16"/>
+        <BarWidth value="130"/>
```

Comment:
Ultralisk: Reduced size by 12.5%
- reduced hp bar width from 160 to 130 (-18.75%)
- reduced hp bar's offset from 30 to 16 (-46.67%)

The hp bar's offset change looks unexpectedly large, but it is alright when tested.
![Screenshot2022-12-12 19_19_56](https://user-images.githubusercontent.com/5763784/207127564-7ff33092-ce59-4809-8d4b-e24e5ca09580.jpg)
![Screenshot2022-12-12 19_20_23](https://user-images.githubusercontent.com/5763784/207127548-fb4b1f8d-6885-477f-b411-aa8cde947a7e.jpg)